cmd/infra/powervs/create.go

```diff
@@ -1554,10 +1560,14 @@ func (infra *Infra) createTransitGateway(ctx context.Context, options *CreateInf
 	}
 
 	log(options.InfraID).Info("Creating Transit Gateway ...")
+	vpcRegion, err := regionutils.VPCRegionForPowerVSRegion(options.Region)
```

Comment:
Please add a comment here on what we are trying to do!

Reply:
I think you can just use `IsGlobalRoutingRequiredForTG()` to decide the global routing.
```suggestion
	globalRouting := regionutils. IsGlobalRoutingRequiredForTG(options.Region, options.VPCRegion)
```